core/src/test/java/com/linecorp/armeria/server/http/HttpServerTest.java

```diff
@@ -394,6 +400,16 @@ public void reset() {
         clientMaxResponseLength = MAX_CONTENT_LENGTH;
     }
 
+    @After
+    public void clearRequestLogs() {
+        try {
+            await().atMost(10, TimeUnit.SECONDS).until(() -> pendingRequestLogs.get() == 0);
```

Comment:
10 secs is the default so no need to set atMost (we never do elsewhere either to remove verbosity)